cmd/openshift-tests/openshift-tests.go

```diff
@@ -278,4 +278,5 @@ func bindOptions(opt *testginkgo.Options, flags *pflag.FlagSet) {
 	flags.IntVar(&opt.Count, "count", opt.Count, "Run each test a specified number of times. Defaults to 1 or the suite's preferred value.")
 	flags.DurationVar(&opt.Timeout, "timeout", opt.Timeout, "Set the maximum time a test can run before being aborted. This is read from the suite by default, but will be 10 minutes otherwise.")
 	flags.BoolVar(&opt.IncludeSuccessOutput, "include-success", opt.IncludeSuccessOutput, "Print output from successful tests.")
+	flags.IntVar(&opt.Parallelism, "parallelism", opt.Parallelism, "Number of tests running in parallel. 0 defaults to test suite recommended value, which is different in each suite.")
```

Comment:
`--max-parallel-tests` to be consistent with other places where we limit overall throughput.

Reply:
Fixed, PTAL